backend/packages/Upgrade/src/api/services/ExperimentAssignmentService.ts

```diff
@@ -267,7 +267,7 @@ export class ExperimentAssignmentService {
         },
       });
 
-      if (experimentDecisionPoint.length) {
+      if (experimentDecisionPoint.length && experimentId !== '') {
         let selectedExperimentDP = experimentDecisionPoint.filter( dp => dp.experiment.id === experimentId);
```

Comment:
Just wondering if it's a better solution than `let selectedExperimentDP = experimentDecisionPoint.filter( dp => dp.experiment.id === experimentId || experimentId === '');` which was what I did to not see the error.

Reply:
Got it, thank you!